quizzes/08_spread.js

```diff
@@ -0,0 +1,83 @@
+console.log(
+  // getMax([7, 8, 5, 9, 20, 7])
```

Comment:
Yes absolutely! I am also on the gitter chat room of this repo. Kindly feel free to tag me there if you need to communicate anything that is not better suited in chat 😅